src/controllers/handlers.ts

```diff
@@ -148,6 +148,16 @@ export const estateMapPngRequestHandler = async (context: {
     const selected = Object.values(tiles).filter(
       (tile) => tile.estateId && tile.estateId === estateId
     )
+    if (selected.length === 0) {
+      const  headers: Record<string, string> = {
```

Comment:
i had to create this here and type it as `Record<string, string>` otherwise it would create some issues with the typings (because this is the only `return` with the key `location` in this handler).